content/rs/references/rest-api/objects/state-machine.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: State Machine object
+linkTitle: state-machine
+description: An object that represents state machines.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+
+A `state-machine` object tracks the creation of a database on a Redis Enterprise cluster. When you create a database on a Redis Enterprise cluster, Redis Enterprise creates a state machine that tracks the status of the database creation process.
+
+A state machine contains the following attributes:
+
+| Name        | Type/Value | Description |
+|-------------|------------|-------------|
+| action_uid  | string     | A globally unique identifier of the action |
+| object_name | string     | Name of the object being manipulated by the state-machine |
+| status      | pending    | Requested state machine has not started |
+|             | active     | State machine is currently running |
+|             | completed  | Operation complete |
+|             | failed     | Operation or State Machine failed |
```

Comment:
consider using `state-machine` here as well. just checking: this and the next two statuses don't have a name?

Reply:
I'm using the table here similarly to the Action object - https://docs.redis.com/staging/jira-doc-1160/rs/references/rest-api/objects/action/ . The "status" can be either pending, active, completed, or failed. I'm using the "Description" column to describe each state.